learn/advanced/filtering_and_faceted_search.md

```diff
@@ -102,10 +102,14 @@ The [`GET` route of the search endpoint](/reference/api/search.md#search-in-an-i
 
 String expressions combine conditions using the following filter operators and parentheses:
 
-- `NOT` only returns documents that do not satisfy a condition : `NOT genres = horror`
+- `NOT` returns all documents that do not satisfy a condition. The expression `NOT genres = horror` returns all documents whose `genres` do not contain `horror` and all documents missing a `genres` field
 - `AND` operates by connecting two conditions and only returns documents that satisfy both of them: `genres = horror AND director = 'Jordan Peele'`
 - `OR` connects two conditions and returns results that satisfy at least one of them: `genres = horror OR genres = comedy`
 - `TO` is equivalent to `>= AND <=`. The expression `release_date 795484800 TO 972129600` translates to `release_date >= 795484800 AND release_date <= 972129600`
+- `IN [valueA, valueB, …, valueN]` selects all documents whose chosen field contains at least one of the specified values. The expression `genres IN [horror, comedy]` returns all documents whose `genres` includes either `horror`, `comedy`, or both
```

Comment:
"The expression `genres IN [horror, comedy]` returns all documents whose `genres` includes either `horror`, `comedy`, or both"
This does cover it, but do you think it would be helpful to add that:
```
attribute IN[value1, value2, value3,]
attribute = value1 OR attribute = value2 OR attribute = value3
```
are the same

Reply:
Same as below: we'll address this post v0.29, when rewriting the guide.